constellations/gemini-admin/src/home/home.js

```diff
@@ -2,107 +2,169 @@
 require("./home.sass");
 import React from "react";
 import API from "../api.js";
-import { Link } from "react-router-dom";
+import { HomeTabSectionClasses } from "./homeClasses.js";
+import { HomeTabSectionUsers } from "./homeUsers.js";
+import { HomeTabSectionRegistrations } from "./homeRegistrations.js";
+import { HomeTabSectionAccounts } from "./homeAccounts.js";
+
+const sectionDisplayNames = {
+    class: "Unpublished Classes",
+    user: "New Users",
+    registration: "New Registrations",
+    unpaid: "Unpaid Accounts",
+};
 
 export class HomePage extends React.Component {
     state = {
-        classes: [],
+        currentSection: "class",
```

Comment:
when we use constants, it's best to define constant variables at the top of the file.

So at the top I would have:

const TAB_CLASSES = "classes";
const TAB_REGISTRATIONS = "registrations";
const TAB_USERS = "users";
const TAB_UNPAID = "unpaid";
then here use: selectedTab: TAB_CLASSES

It's better to do things this way because you force tab selection to use "typed" variables instead of a value where you are prone to typos.
For example, if you had currentSection: "clases", you might not notice this problem until your tab selection doesn't work. However, if you did selectedTab: TAB_CLASES you would get a compile error and immediately alert you!